src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/VelgPeriode.tsx

```diff
@@ -35,23 +32,15 @@ const StyledLabel = styled(Label)`
     margin-right: 0.5rem;
 `;
 
-const MarginSkjemaGruppe = styled(SkjemaGruppe)`
+const MarginFieldset = styled(Fieldset)`
     margin-bottom: 1rem !important;
 `;
 
 const FlexDiv = styled.div`
-    width: 23rem;
     display: flex;
-    justify-content: space-between;
-    & .lese-element {
-        width: 50%;
-    }
-    .skjemaelement__label {
-        color: ${navFarger.navMorkGra};
-        font-size: 16px;
-        font-weight: normal;
-        height: 22px;
-        line-height: 22px;
+    gap: 1.125rem;
+    & > * {
+        min-width: 11rem;
```

Comment:
Dette er for å ha jevn plassering av fom og tom uavhengig av om det er lesevisning